packages/docs/app/layout.tsx

```
@@ -0,0 +1,43 @@
+import { InkeepBubble } from "@/components/inkeep-bubble";
+import InkeepSearchBox from "@/components/inkeep-search";
+
+import "./global.css";
+import Scroller from "@/components/scroller";
+import { Analytics } from "@vercel/analytics/react";
+import { Banner } from "fumadocs-ui/components/banner";
```

Biome (1.9.4)

[error] This import is unused.

Unused imports might be the result of an incomplete refactoring.

(lint/correctness/noUnusedImports)